lib/matplotlib/image.py

```diff
@@ -214,17 +214,16 @@ def _resample(
     return out
 
 
-def _rgb_to_rgba(A):
+def _rgb_to_rgba(A, alpha=None):
     """
     Convert an RGB image to RGBA, as required by the image resample C++
     extension.
     """
     rgba = np.zeros((A.shape[0], A.shape[1], 4), dtype=A.dtype)
+    if alpha is None or np.ndim(alpha) == 0:
```

Comment:
what input is `np.ndim(alpha) == 0` addressing?

Reply:
Scalar alpha input should remaing unaffected by new code. This ensures just that